mirage/config.js

```diff
@@ -285,7 +285,10 @@ export default function (config) {
     issns: ['Print:0003-2654', 'Online:1364-5528'],
     journalName: 'The Analyst',
     nlmta: 'Analyst',
+    pmcParticipation: 'B',
```

Comment:
We need to add this explicitly because otherwise, this default https://github.com/eclipse-pass/pass-ui/pull/1224/files#diff-cd49b43c6fe4ed8bd896068427733aa3dab8fb10d917ab6bf67a0c325b0a6811R7 in the factory overrrides it.